shoop_setup_utils/commands.py

```diff
@@ -82,9 +86,29 @@ class BuildProductionResourcesCommand(BuildResourcesCommand):
     clean = True
 
 
+class BuildTranslationsCommand(distutils.core.Command):
+    command_name = 'build_translations'
+    description = 'compile message catalogs to MO files via Django compilemessages'
+    user_options = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        os.chdir("shoop")
+        settings.configure()
```

Comment:
Management command has the advantage that DJANGO_SETTINGS_MODULE is set. Compiling messages to mo does not need the settings module being set, but makemessages needs it, because the django_jinja extensions read settings (for some reason). Having the makemessages part in setup.py would be another option, but that would require setting DJANGO_SETTINGS_MODULE to some value, which would be `shoop_workbench.settings` for shoop, but there would be no way to guess it for addons.